packages/apollo/src/utils/errors.ts

```diff
@@ -0,0 +1,9 @@
+import { CLIError } from "@oclif/errors";
+
+const errorMessage = [
+  "No graph (i.e. service) found to link to Apollo Graph Manager.",
+  "In order to run this command, please provide a graph ID using the 'apollo.config.js' file.",
+  "\n\nFor more information on configuring the Apollo CLI, please go to",
+  "https://www.apollographql.com/docs/devtools/apollo-config/"
```

Comment:
```suggestion
  "https://go.apollo.dev/t/config"
```

Can you use the short link here instead just in case we change that in the future? 😊